### DISCUSSION TITLE
d5ff0d1 continuous

### DISCUSSION THREAD

**Discussion by Terracotta-62** [artisan-roaster-scope/artisan#1217] [General]
Summary Box Loss of information content

**Comment by MAKOMO** [comment]
Could you provide more information on this. A screen shot and a profile/settings would help to understand what you are reporting. Thanks!

**Comment by Terracotta-62** [comment]
![d30278c8c08c0c7bbe91fe51f0472ed](https://github.com/artisan-roaster-scope/artisan/assets/116416332/cc24664f-d7e5-4282-a73a-68d80a6fab98)

**Reply by Terracotta-62** [reply]
Only in this version

**Reply by MAKOMO** [reply]
Not for me. Could please attach your settings and that profile to allow me to reproduce this? I assume that this is data dependent and caused by some character encoding issue. Thanks!

**Comment by Terracotta-62** [comment]
The cause of this occurrence is unknown, and the configuration file is reloaded. The information in the summary box is displayed normally.
Thanks !  
![97e88d20ca29b56f0c633a038272f2a](https://github.com/artisan-roaster-scope/artisan/assets/116416332/2ab402a1-ce4a-4187-a10f-8b813880180d)
